platform/src/ActivityManager.js

```diff
@@ -346,15 +346,40 @@ class ActivityManager {
         return null;
     }
 
+    /**
+     * Interpolate someString using information from session storage.
+     * 
+     * @param {*} someString the string to be changed
+     */
+    interpolate(someString) {
+        let result = someString;
+
+        // console.log (`Asked to interpolate ${someString}.`);
```

Comment:
The commented out code for console logging should be removed. 

A single summary could be logged when a replacement is made so the values and their replacements can be seen.

Reply:
Agreed. I have removed the commented-out code. I haven't put in an active log statement as it seems a lot of logging for very little use...